lib/builder/builder.go

```diff
@@ -260,10 +260,7 @@ func (b *Builder) SyncPackageCache(ctx context.Context, runtimeVersion semver.Ve
 	for _, runtimeVersion := range append([]semver.Version{runtimeVersion}, intermediateVersions...) {
 		b.NextStep("Syncing packages for %v", runtimeVersion)
 		if err := b.syncPackageCache(ctx, runtimeVersion, syncer, apps); err != nil {
-			if trace.IsNotFound(err) {
-				return trace.NotFound("runtime version %v not found", runtimeVersion)
-			}
-			return trace.Wrap(err)
+			return trace.Wrap(err, "failed to sync packages for runtime version %v", runtimeVersion)
```

Comment:
This also fixes an issue with shadowing package errors from the user using `tele build`.